packages/oidc4vc/lib/src/oidc4vc.dart

```diff
@@ -1603,7 +1630,7 @@ class OIDC4VC {
       'aud': tokenParameters.audience,
       'exp': iat + 1000,
       'sub': tokenParameters.did,
-      'iss': tokenParameters.did,
+      //'iss': tokenParameters.did,
```

Comment:
Why removing iss for all vp tokens ? Most presentation are failing without iss. @bibash28

Reply:
yes we should get is back..